client/gutenberg/extensions/tiled-gallery/tiled-grid/constrained-array-rounding/index.js

```diff
@@ -84,9 +78,5 @@ export const getRoundedConstrainedArray = ( bound_array, sum_value = false ) =>
 
 	bound_array_int = adjust_constrained_array( bound_array_int, diff_sum );
 
-	const bound_array_fin = map( bound_array_int, property( 'floor' ) );
-
-	return bound_array_fin;
-	// Combines keys and values into an object using lodash.zipObject
-	//return zipObject( keys, bound_array_fin );
+	return map( bound_array_int, property( 'floor' ) );
```

Comment:
I think with lodash we can write `map( arr, property( 'floor' ) )` can be written `map( arr, 'floor' )`